assets/css/components/_pagination.scss

```diff
@@ -0,0 +1,56 @@
+.pagination {
+  display: block;
+  list-style: none;
+  margin-top: 10px;
+  padding: 0;
+  text-align: center;
+
+  .pagination-list {
+    display: inline-block;
+  }
+
+  .prev-next {
+    height: rem(53);
+    margin: 0 rem(20);
+  }
+
+  li {
+    display: inline;
+    text-align: center;
+
+    a.nuxt-link-active {
+      background-color: #ffa100;
+    }
+  }
+
+  .pagination-link {
+    background-color: #f7c12c;
```

Comment:
Please use color variables here.

Reply:
These are being used as place holders until @lliskovoi comes up with a design for the pagination. Should I still change them to cover variables?